### PR TITLE
Collective Voice draft article

### PR DESCRIPTION
**Pull request by knowmad**

### CODE REVIEW

**Comment by knowmad** [comment]
@cromedome - the conflict in README is probably me changing my name to my knowmad handle.

File: pending/13-collective-voice.pod [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+It took a bit of trial and error to get into the hang of building interfaces up from simple utility classes, but I am hooked! It eliminates the need for writing custom CSS using id's and nested classes. The end result is less coding and more succinct designs. Check out an example at L<tailwindcss.com/docs/utility-first|https://tailwindcss.com/docs/utility-first>.
+
+One of the downsides of using minimalist frameworks is that you often need to pull in other libraries to provide additional support. In this case, I needed a Javascript library for the modal dialogs and form validation.
+
+Rather than pulling in a fat library like jQuery for this purpose, I learned that many Tailwind developers were opting for a lightweight alternative called Alipine JS. This library offers the reactive and declarative nature of big frameworks like Vue or React at a much lower resource cost.
```
Comment (cromedome) [inline pending/13-collective-voice.pod:47]: `s/Alipine/Alpine;`

**Comment by cromedome** [comment]
One typo, easy fix. Good description of the business case.

Should have put the second article stub in a second PR. Not a huge deal :) 

👍

**Comment by cromedome** [comment]
I fixed the typo and merged to master.

**Pull request closed without merging**